understanding/understanding-act-rules.html

```diff
@@ -51,95 +49,72 @@
                </p>
                		
-               <p><a href="conformance">Understanding Conformance</a> provides related information, including on <a href="conformance#accessibility-support">understanding accessibility support</a>.
+               <p><a href="conformance">適合を理解する</a>は、<a href="conformance#accessibility-support">アクセシビリティサポートを理解する</a>を含む、関連する情報を提供する。
                </p>
                		
                		
                <section id="test-rules-rules-are-informative">
                   			
-                  <h2>Test Rules Rules are Informative</h2>
+                  <h2>テストルールは参考情報である</h2>
                   			
-                  <p><em>Test Rules are informative — that means they are not required for determining conformance.
-                        The basis for determining conformance to WCAG is the success criteria from the WCAG
-                        standard — not the Test Rules.</em></p>
+                  <p><em>テストルールは参考情報である。つまり、適合を判断するために必要ではないことを意味する。WCAG への適合を判断するための基礎は、WCAG テストルールではなく、WCAG 標準の達成基準である。</em></p>
                   			
-                  <p>While W3C's <a href="https://www.w3.org/WAI/standards-guidelines/act/rules/">List of Test Rules for WCAG 2</a> are reviewed by the W3C Accessibility Guidelines Working Group (AGWG), they are not
-                     vetted to the same degree as the W3C Web Content Accessibility Guidelines (WCAG) standard
-                     (called <em>W3C Recommendation</em>). The WCAG standard is the normative reference on determining conformance.
+                  <p>W3C の <a href="https://www.w3.org/WAI/standards-guidelines/act/rules/">List of Test Rules for WCAG 2</a> は、W3C の Accessibility Guidelines Working Group (AGWG) によってレビューされているが、W3C Web Content Accessibility Guidelines (WCAG) 標準 (<em>W3C 勧告</em>と呼ばれる) と同程度に精査されていない。WCAG 規格は、適合を判断するための規範的な基準である。
                   </p>
                   		
                </section>
                		
                		
                <section id="test-rules-are-partial-checks">
                   			
-                  <h2>Test Rules are Partial Checks</h2>
+                  <h2>テストルールは部分的なチェックである</h2>
                   			
-                  <p>Test Rules typically check specific aspects of WCAG success criteria. For example,
-                     that a table cell has a header rather than the entire WCAG 2.2 success criterion 1.3.1
-                     "Info and Relationships", which applies to many more information structures on a web
-                     page. In fact, this example rule would not even check the validity of the table header,
-                     only if the header exists for a given table cell.
+                  <p>テストルールは通常、WCAG 達成基準の特定の側面をチェックする。例えば、テーブルセルには、WCAG 2.2 の全体の達成基準 1.3.1「情報及び関係性」ではない、ヘッダーがある。これは、ウェブページ上のより多くの情報構造に適用される。実際、このサンプルのルールは、特定のテーブルセルにヘッダーが存在する場合に限り、テーブルヘッダーの有効性をチェックさえしない。
```

Comment:
これなんかすごい誤訳のような感じがしますね。多分こういう感じなのではないかと思いますが、どうでしょうか。
(要するに、「table headerのテストをやったからと言って達成基準1.3.1を全部満たしていることを意味するわけではない」ということを難しく言っている……)

```suggestion
                  <p>テストルールは通常、WCAG 達成基準の特定の側面をチェックする。例えば、テーブルセルにヘッダーがあることは、WCAG 2.2 の達成基準 1.3.1「情報及び関係性」の範疇であるが、この基準はウェブページ上のより多くの情報構造に適用される。実際、このサンプルのルールは、テーブルヘッダーの有効性さえチェックせず、特定のテーブルセルに対するヘッダーが存在するかどうかチェックするだけである。
```

Reply:
いただいた文面を取り込み、many moreを「より多く」から「はるかに多く」としてみました。